src/components/KCheckbox/KCheckbox.vue

```diff
@@ -62,59 +62,36 @@
 </template>
 
 <script lang="ts" setup>
-import type { PropType } from 'vue'
-import { computed, useAttrs, useId, useSlots } from 'vue'
-import type { LabelAttributes } from '@/types'
+import { computed, useAttrs, useId } from 'vue'
+import type { CheckboxProps, CheckboxEmits, CheckboxSlots } from '@/types'
 import KLabel from '@/components/KLabel/KLabel.vue'
 import { CheckSmallIcon, IndeterminateSmallIcon } from '@kong/icons'
 import { KUI_ICON_SIZE_40 } from '@kong/design-tokens'
 
-const props = defineProps({
-  modelValue: {
-    type: Boolean,
-    default: false,
-    required: true,
-  },
-  label: {
-    type: String,
-    default: '',
-  },
-  labelAttributes: {
-    type: Object as PropType<LabelAttributes>,
-    default: () => ({}),
-    validator: (value: LabelAttributes): boolean => {
-      if (value.help) {
-        console.warn('KCheckbox: `help` property of `labelAttributes` prop is deprecated. Please use `info` prop instead. See the migration guide for more details: https://kongponents.konghq.com/guide/migrating-to-version-9.html#klabel')
-      }
-
-      return true
-    },
-  },
-  description: {
-    type: String,
-    default: '',
-  },
-  error: {
-    type: Boolean,
-    default: false,
-  },
+defineOptions({
+  inheritAttrs: false,
```

Comment:
@adamdehaven I removed the trailing `script` section and moved it here, just in case you missed this.

Reply:
I was referencing the deprecation log

Reply:
@adamdehaven thanks, those were added in the latest commit :bow: